### PR TITLE
Docker image updates

### PR DESCRIPTION
**Pull request by lloesche**
- integrates keepercore into Dockerfile
- include static content with core package
- migrate Alpine -> Debian
- Add ArangoDB to image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,23 @@
-FROM python:3.9-alpine AS build-env
+FROM debian:stable-slim as build-env
```
Comment (aquamatthias) [inline Dockerfile:1]: 👍

File: keepercore/MANIFEST.in [hunk 2]
```diff
@@ -1,2 +1 @@
-include requirements-dev.txt
-include requirements.txt
```
Comment (aquamatthias) [inline keepercore/MANIFEST.in:2]: I remember that I had to add those files manually here. Can't remember why...

**Review by aquamatthias — APPROVED** [review]
lgtm. thanks!

File: keepercore/MANIFEST.in (hunk 2 above)
Comment (lloesche) [reply keepercore/MANIFEST.in]: Ah right, I didn't see that there was an existing MANIFEST and just replaced it with the static content. Those two includes are required for the source distribution to be able to include the txt files in `setup.py`.

**Pull request merged**